etc/kayobe/environments/ci-aio/controllers.yml

```diff
@@ -5,3 +5,8 @@
 # User with which to access the controllers via SSH during bootstrap, in order
 # to setup the Kayobe user account. Default is {{ os_distribution }}.
 controller_bootstrap_user: "{{ os_distribution if os_distribution == 'ubuntu' else 'cloud-user' }}"
+
+# Controller lvm configuration. See intentory/group_vars/controllers/lvm.yml
+# for the exact configuration.
+controller_lvm_groups:
+  - "{{ stackhpc_lvm_group_rootvg }}"
```

Comment:
This will make CI jobs fail if the image is not using LVM

Reply:
I'll hold off until we're using LVM images in CI then